yt/yt/ytlib/api/native/client_authentication_impl.cpp

```diff
@@ -15,12 +16,23 @@ using namespace NYTree;
 
 ////////////////////////////////////////////////////////////////////////////////
 
+/*
+ * User-created cypress tokens follow the format "ytct-{}-{}".
+ * First {} is 4 hexadecimal characters and may be saved and revealed to the user.
+ * Second {} is 32 hexadecimal characters, private, and should not be saved in the system.
+ */
+constexpr TStringBuf CypressTokenPrefixRegex = "ytct-[0-9a-f]{4}-";
+constexpr int        CypressTokenPrefixLength = 10; // "ytct-abcd-"
```

Comment:
We do not ident names in a way like this, just ` constexpr int CypressTokenPrefixLength = 10`